src/components/CompletingParty/CompletingParty.vue

```diff
@@ -204,8 +204,8 @@ export default class CompletingParty extends Vue {
 
   private get getCompletingPartyName (): string {
     return (
-      this.currentCompletingParty.firstName +
-      this.currentCompletingParty.middleName +
+      this.currentCompletingParty.firstName + ' ' +
+      this.currentCompletingParty.middleName + ' ' +
```

Comment:
The spaces need to be conditional.

Reply:
Or you could `trim()` the result and then check if it's empty.

Reply:
doh, of course

Reply:
Updated to template with conditional middle